manifests/repo.pp

```diff
@@ -1,49 +1,53 @@
-# private class
-class sensu::repo {
+# @summary Private class to manage sensu repository resources
+# @api private
+#
+class sensu::repo (
+  $manage_repo = $sensu::manage_repo
+) inherits sensu {
 
-  case $facts['os']['family'] {
-    'RedHat': {
-      if $facts['os']['name'] == 'Amazon' {
-        if $facts['os']['release']['major'] =~ /^201\d$/ {
-          $repo_release = '6'
+  if any2bool($manage_repo) {
```

Comment:
Don't need `any2bool` since `$manage_repo` should have data type `Boolean`.

Reply:
it makes sense: remove any2bool, since it's always bool

Reply:
done